app/src/main/res/layout/act_nave_host.xml

```diff
@@ -1,18 +1,32 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
The file name should be act_nav_host there is a typo in that.

Reply:
Done!